modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/Cookie.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const CONSENT_TYPES = {
+	FUNCTIONAL: 'CONSENT_TYPE_FUNCTIONAL',
+	NECESSARY: 'CONSENT_TYPE_NECESSARY',
+	PERFORMANCE: 'CONSENT_TYPE_PERFORMANCE',
+	PERSONALIZATION: 'CONSENT_TYPE_PERSONALIZATION',
+};
+
+const generateCookie = (name, value, options = {}) =>
+	`${name}=${value}; ${Object.entries(options)
+		.map(([key, value]) => {
+			if (key === 'secure') {
+				return value ? 'secure' : '';
+			}
+
+			return `${key}=${value}`;
+		})
+		.join('; ')}`.trim();
+
+const Cookie = {
```

Comment:
I'm not a 100% sure of this, but I believe that we are trying to use functions instead of classes these days in JS. I have nothing against classes just thinking about consistency. 

Also, you should be able to `export default const ...` directly here instead of below the declaration.

Reply:
> I'm not a 100% sure of this, but I believe that we are trying to use functions instead of classes these days in JS. I have nothing against classes just thinking about consistency.

Thanks for the feedback! One question just to check I got it right, should I follow the same pattern as in [Liferay.Util.Session](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js)? So export each function individually and then group them in [global.es.js](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js#L333-L336)? In that case should I rename this to `cookie.es.js`? 

> Also, you should be able to export default const ... directly here instead of below the declaration

I agree that should be an option, but [you cannot `export default const/let/var`](https://stackoverflow.com/a/36261387) 😞 
![image](https://user-images.githubusercontent.com/6642927/172149957-69f7629c-13f0-41d0-bc65-b54199c67574.png)

Reply:
No problem, I don't feel strongly about it 😄 I'll change it to follow @carloslancha's snippet

Reply:
So I followed @carloslancha's approach, but since the Cookie utility needs to be available from `Liferay.Util.Cookie` (e.g. `Liferay.Util.Cookie.set`) and should also be importable from `frontend-js-web` as `import {setCookie} from 'frontend-js-web'`, I also added individual exports for `get|set|removeCookie` from `cookie.js` aside from the default export with the short names. 

I'm not 100% convinced, but:
* `Liferay.Util.Cookie.setCookie` seems redundant
* It looks like `frontend-js-web` exports each individual function instead of objects, so it's the only way I've found to achieve this behaviour without regrouping and renaming for `Liferay.Util.Cookie` in `global.es.js`

Any feedback appreciated ❤️ @kresimir-coko @carloslancha